src/crimson/os/seastore/seastore_types.h

```diff
@@ -1739,8 +1732,9 @@ struct record_group_size_t {
 
   void account(const record_size_t& rsize,
                extent_len_t block_size);
+
+  auto operator<=>(const record_group_size_t &) const = default;
```

Comment:
Ah, forget to point out that this is also == operator only.

Reply:
fixed and repushed.